### DISCUSSION TITLE
Why is v1 still the recommended version in the docs, when v1 is shutting down?

### DISCUSSION THREAD

**Discussion by paintoshi** [WalletConnect/walletconnect-monorepo#2707] [Developer Support]
Version 1 is shutting down the end of June. Yet the recommended option in your react example is still v1 and not v2.
How come? Or is this version something else?

https://docs.walletconnect.com/2.0/web/web3modal/react/wagmi/installation

```
const wagmiConfig = createConfig({
  autoConnect: true,
  connectors: w3mConnectors({ projectId, version: 1, chains }),
  publicClient
})
```

**Comment by glitch-txs** [comment]
Thanks for pointing out

**Comment by glitch-txs** [comment]
Btw would you like opening a PR on the docs repo? I can notify the team

**Reply by paintoshi** [reply]
Sure: https://github.com/WalletConnect/walletconnect-docs/pull/785

**Reply by glitch-txs** [reply]
Thanks! Sending to the team 🙂
